stochastic/continuous/mixed_poisson.py

```diff
@@ -0,0 +1,85 @@
+"""Mixed poisson processes."""
+
+from stochastic.base import Checks
+from stochastic.continuous import PoissonProcess
+
+class MixedPoissonProcess(PoissonProcess):
+    r"""Mixed poisson process. Inherits from the PoissonProcess class.
+
+    A mixed poisson process is a Poisson process for which the rate is a random variate, 
+    a sample taken from a random distribution. On every call of the sample, a new random rate is generated.
+    A Poisson process with rate :math:`\lambda` is a count of occurrences of
+    i.i.d. exponential random variables with mean :math:`1/\lambda`. This class
+    generates samples of times for which cumulative exponential random variables occur. 
+
+    :param function rate_func: random distribution of the rate :math:`\lambda`
+    :param rate_args: arguments to input into the rate_func function
+    :param rate_kwargs: keyword arguments to input into the rate_func function
+    """
+
+    def __init__(self, rate_func,*rate_args,**rate_kwargs):
```

Comment:
Space after commas.

rate_args and rate_kwargs should be passed as single objects, not unpacked.